scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -504,6 +517,34 @@ public void execute(Boolean value) {
                                 schedResumeButton,
                                 schedPauseButton,
                                 schedKillButton);
+        // Adding tooltips on Admin actions
+        ListGridField titleFieldDefaults = adminMenu.getTitleFieldDefaults();
+        titleFieldDefaults.setShowHover(true);
+        titleFieldDefaults.setHoverCustomizer(new HoverCustomizer() {
+            @Override
+            public String hoverHTML(Object value, ListGridRecord record, int rowNum, int colNum) {
+
+                if (value.toString().equalsIgnoreCase(START))
```

Comment:
if you want you can replace sequence of if-esle-if with switch.
```
switch (value.toString().toUpperCase()) {
case START:....
..
}

```

Reply:
Since START == "Start", this will not work..